doc/dependencies.md

```diff
@@ -23,7 +23,7 @@ These are the dependencies currently used by Bitcoin Core. You can find instruct
 | protobuf | [2.6.1](https://github.com/google/protobuf/releases) |  | No |  |  |
 | Python (tests) |  | [3.4](https://www.python.org/downloads) |  |  |  |
 | qrencode | [3.4.4](https://fukuchi.org/works/qrencode) |  | No |  |  |
-| Qt | [5.9.7](https://download.qt.io/official_releases/qt/) | [5.2](https://github.com/bitcoin/bitcoin/pull/14725) | No |  |  |
+| Qt | [5.9.7](https://download.qt.io/official_releases/qt/) | 5.5.1 | No |  |  |
```

Comment:
Could you (assuming it is merged) link to this PR as a reference? It's useful when referring to the docs to be able to jump straight to the PR/discussion that bumped a dependency.